### PR TITLE
Shim elk: remove bends

### LINKED ISSUE #1030: shim elk: Reduce edge bends
These multiple bends seem easily reducible to one or zero bend, but ELK does not seem to have a setting to do that. I've tried many options and searching issues. I think we can apply some after-cream here

Screen Shot 2023-03-14 at 1 09 36 PM

### PR DESCRIPTION
**Pull request by alixander**
ELK no longer looks like it has parkinsons. Going to make it the default in next release.

### Before
Screen Shot 2023-03-14 at 5 42 09 PM

### After
Screen Shot 2023-03-14 at 5 41 42 PM

screencapture-file-Users-alexanderwang-dev-alixander-d2-e2etests-out-e2e-report-html-2023-03-14-18_37_41
screencapture-file-Users-alexanderwang-dev-alixander-d2-e2etests-out-e2e-report-html-2023-03-14-18_37_41-2

no runtime diffs btw

Screen Shot 2023-03-14 at 5 45 59 PM

closes #1030

### CODE REVIEW

File: e2etests/testdata/stable/n22_e32/elk/sketch.exp.svg [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" d2Version="v0.2.4-HEAD" preserveAspectRatio="xMinYMin meet" viewBox="0 0 897 1230"><svg id="d2-svg" class="d2-3394510602" width="897" height="1230" viewBox="11 11 897 1230"><rect x="11.000000" y="11.000000" width="897.000000" height="1230.000000" rx="0.000000" class=" fill-N7" stroke-width="0" /><style type="text/css"><![CDATA[
-.d2-3394510602 .text-bold {
-	font-family: "d2-3394510602-font-bold";
+<?xml version="1.0" encoding="utf-8"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" d2Version="v0.2.4-HEAD" preserveAspectRatio="xMinYMin meet" viewBox="0 0 897 1230"><svg id="d2-svg" class="d2-1920563192" width="897" height="1230" viewBox="11 11 897 1230"><rect x="11.000000" y="11.000000" width="897.000000" height="1230.000000" rx="0.000000" class=" fill-N7" stroke-width="0" /><style type="text/css"><![CDATA[
```
Comment (alixander) [inline e2etests/testdata/stable/n22_e32/elk/sketch.exp.svg:1]: oop, a bug here it looks like

**Comment by gavin-ts** [comment]
#1026 + a rebase to update new test should fix CI errors

File: d2layouts/d2elklayout/layout.go [hunk 2]
```diff
@@ -442,5 +443,270 @@ func Layout(ctx context.Context, g *d2graph.Graph, opts *ConfigurableOpts) (err
...
+			if newOverlapsCount > oldOverlapsCount {
+				continue
+			}
+
+			if newCloseOverlapsCount > oldCloseOverlapsCount {
+				continue
+			}
+			if newTouchingCount > oldTouchingCount {
+				continue
+			}
```
Comment (gavin-ts) [inline d2layouts/d2elklayout/layout.go:537]: it seems like we only need the overlaps count since it covers close overlaps and touching

**Review by gavin-ts — APPROVED** [review]

File: d2layouts/d2elklayout/layout.go (hunk 2 above)
Comment (alixander) [reply d2layouts/d2elklayout/layout.go:537]: we don't want it to get progressively worse

**Pull request merged**